### PR TITLE
Add thousands separator to crashes and locations lists

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/15866

## Testing
**URL to test:** 
[Netlify
](https://deploy-preview-1404--atd-vze-staging.netlify.app/#/dashboard)
**Steps to test:**
1. Check out the crashes and locations page, observe and admire the comma separators making numbers in the thousands more readable :sparkles:

---
#### Ship list
- [x] Check migrations for any conflicts with latest migrations in master branch
- [x] Confirm Hasura role permissions for necessary access
- [x] Code reviewed
- [x] Product manager approved

### CODE REVIEW

File: atd-vze/src/queries/gqlAbstract.js [hunk 1]
```diff
@@ -307,7 +307,7 @@ gqlAbstractTableAggregateName (
         return `${dateValue}`;
       }
       case "currency": {
-        return `$${value.toLocaleString()}`;
+        return `$${parseFloat(value).toLocaleString()}`;
```
Comment (roseeichelmann) [inline atd-vze/src/queries/gqlAbstract.js:310]: this wasn't working previously cuz `value` is a string type so first we gotta parse it and return as a float :p

**Review by frankhereford — APPROVED** [review]
I am observing and admiring the new commas! Wouldn't change a thing, this is great Rose. Thank you!

**Review by mddilley — APPROVED** [review]
Looks great! Thanks for improving all these numbers - they look so nice in the app! ✨

File: atd-vze/src/queries/gqlAbstract.js (hunk 1 above)
Comment (roseeichelmann) [reply atd-vze/src/queries/gqlAbstract.js]: ah good idea! just pushed that up. yeah wrangling helpers would be good!

**Review by johnclary — APPROVED** [review]
ahh—this is so much better. thank you! 🚢 🚢 🚢 🚢 🚢

**Pull request merged**